php/classes/class-all-episode-stats.php

```diff
@@ -125,7 +129,8 @@ private function get_all_episode_stats() {
 
 			$episode_stats = array(
 				'episode_name' => $post->post_title,
-				'date'         => date( 'm-d-Y', strtotime( $post->post_date ) ),
+				// updated date format here to fix sort order 
+				'date'         => date( 'Y-m-d', strtotime( $post->post_date ) ), 
```

Comment:
This makes logical sense. However I'm concerned about users who might be used to the original format. Is there a way we can fix the sort order without making this change? I'm thinking if we add a 'date_display' field, and allow it to be filtered, we can show the display date in the same way, but allow users to change that if they prefer.

Reply:
On second thought, it might be preferable to use the WordPress Date Format settings, under the General Options settings.

Reply:
I like the suggestion of using the WordPress Date Format setting. Looking into it.

Reply:
Pushed update to use format in WordPress options.